x-pack/metricbeat/module/gcp/metrics/_meta/docs.asciidoc

```diff
@@ -45,6 +45,11 @@ services under "Google Cloud metrics", but does not work for other services
 (`kubernetes` aka GKE for example).
 This option allow to override the default and specify an arbitrary metric prefix.
 
+* *servicelabel*: This config option is to add the resource label( eg:"resource.label.location") for a 
+   GCP service to filter the data of the service. For a new service which is not supported by default 
+   by Elastic, this label can be set by the user. If not set, tje default servicelabel (resource.label.zone) 
+   will be chosen
```

Comment:
```suggestion
* *location_label*: Use this option to specify the resource label that identifies the location (such as zone or region) for a Google Cloud service when filtering metrics. For example, labels like `resource.labels.zone` or `resource.labels.region` are used by Google Cloud to represent the zone or region of a resource.
```

Reply:
We can go with location label. Lets finalize on the approach we want to take for existing services. Will update the description then